sparkle/src/components/Avatar.tsx

```diff
@@ -144,8 +150,66 @@ export function Avatar({
           {name[0].toUpperCase()}
         </span>
       ) : (
-        <User className="s-h-1/2 s-w-1/2 s-opacity-20" /> // Default icon with size and slate-900 color
+        <User className="s-h-1/2 s-w-1/2 s-opacity-20" />
       )}
     </div>
   );
 }
+
+interface AvatarStackProps {
+  children: ReactNode;
+  maxAvatars?: number;
```

Comment:
I would be in favor of removing the `maxAvatars` logic. I still it's cleaner to slice a proper array of object from `front` to ensure we display the expected number, than to slice here the children react components. WDYT?

Reply:
That why it's a prop is it not?
It allows handling the display of the totall number automatically

Reply:
Yes but that expects we will send the full children components to display only the number specified in `maxAvatars`